tests/acceptance/01_vars/01_basic/CFE-628.cf.sub

```diff
@@ -0,0 +1,14 @@
+bundle agent main 
+{
+  classes:
+    "found_the_file" expression => fileexists( $(this.promise_filename) );
+
+  vars:
+    found_the_file::
```

Comment:
It's not clear to me how this tests a variable reference with nested parenthesis. I only ever see one set of parenthesis.

Reply:
Even reading the bug I don't completely understand the emphasis on "nested parenthesis". It seems the more important distinction is whether the reports promises should print out twice or not, due to multiple passes and the found_the_file class being both true and false.

Does this test currently fail? Should it be marked as soft fail then?

Reply:
No, the test passes.

Ah, the test header was from a copied file. un-related.